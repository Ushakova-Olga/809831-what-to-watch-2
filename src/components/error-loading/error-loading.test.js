import React from "react";
import renderer from "react-test-renderer";
import ErrorLoading from "./error-loading";

it(`Error loading component correctly renders`, () => {
  const tree = renderer.create(<ErrorLoading error={`error`} />).toJSON();
  expect(tree).toMatchSnapshot();
});
