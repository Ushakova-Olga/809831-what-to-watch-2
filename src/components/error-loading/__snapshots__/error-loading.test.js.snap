// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error loading component correctly renders 1`] = `
<h1
  style={
    Object {
      "lineHeight": "2em",
      "marginTop": "200px",
      "textAlign": "center",
    }
  }
>
  Произошла ошибка 
  error
  . Попробуйте загрузить страницу попозже.
</h1>
`;
